### DISCUSSION TITLE
[Support] mwts 的 eslint 以及相关依赖会更新吗？

### DISCUSSION THREAD

**Discussion by Yuliang-Lee** [midwayjs/midway#3930] [Q&A]
### Describe the problem（描述问题）

https://github.com/midwayjs/mwts/pull/16 这个 pr 的 CI 没跑通，官方人员能更新下这个项目吗？

### Midway Versions（Midway 版本）

~

**Comment by waitingsong** [comment]
主要问题是 https://github.com/typescript-eslint/typescript-eslint 还在 v7，不支持 eslint9 。
最新 v8 是 rc 阶段，等正式版了就可以升级了
